### PR TITLE
Update pipeline version to the 1.0.0

### PR DESCRIPTION
**Pull request by JohanWork**
update export PIPELINE_VERSION=0.5.1 to point to 1.0.0 instead of the old version 0.5.1

### CODE REVIEW

File: layouts/shortcodes/pipelines/latest-version.html [hunk 1]
```diff
@@ -1 +1 @@
-0.5.1
\ No newline at end of file
+1.0.0
```
Comment (NikeNano) [inline layouts/shortcodes/pipelines/latest-version.html:1]: Why version 1.0.0 and not the latest?

Comment (JohanWork) [reply layouts/shortcodes/pipelines/latest-version.html:1]: https://github.com/kubeflow/pipelines 1.0.0 seams to be the latest one to me

**Comment by Bobgy** [comment]
master branch is already 1.0.0 https://github.com/kubeflow/website/blob/master/layouts/shortcodes/pipelines/latest-version.html

**Comment by JohanWork** [comment]
> master branch is already 1.0.0 https://github.com/kubeflow/website/blob/master/layouts/shortcodes/pipelines/latest-version.html

sorry your right, got confused since https://www.kubeflow.org/docs/pipelines/installation/standalone-deployment/#deploying-kubeflow-pipelines is still showing 0.5.1 . Why is that @Bobgy ?

**Comment by Bobgy** [comment]
kubeflow.org is not showing master, because everyone is working on kubeflow 1.1

**Comment by JohanWork** [comment]
aa ok thnx!

**Pull request closed without merging**